src/main/java/kr/flab/tradingmarket/domain/product/service/DefaultProductCommandService.java

```diff
@@ -66,4 +66,10 @@ public void modifyProduct(Long productNo, RequestModifyProductDto modifyProduct,
             throw e;
         }
     }
+
+    @Override
+    public void deleteProduct(Long productNo) {
+        Optional<List<ProductImage>> productImages = productService.deleteProduct(productNo);
+        productImages.ifPresent(imageService::deleteProductImages);
```

Comment:
nit : https://stackoverflow.com/questions/26327957/should-java-8-getters-return-optional-type/26328555#26328555